apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/cache/PublicKeyCacheConfig.java

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c)  2018-2019. Apollo Foundation.
+ */
+
+package com.apollocurrency.aplwallet.apl.core.cache;
+
+import com.apollocurrency.aplwallet.apl.util.cache.CacheConfigurator;
+
+public class PublicKeyCacheConfig extends CacheConfigurator {
+
+    public static final String PUBLIC_KEY_CACHE_NAME = "PUBLIC_KEY_CACHE";
+
+    public PublicKeyCacheConfig(int percentCapacity) {
+        super(PUBLIC_KEY_CACHE_NAME, 32, percentCapacity);
```

Comment:
32 - size of public key array, not a size of array object itself

Reply:
yes, I'll resolve it in next PR.

Reply:
I'll add new MemoryUsageCalculator to help calculate memory usage.